css/app.css

```diff
@@ -1,38 +1,48 @@
 @import url('https://fonts.googleapis.com/css2?family=Work+Sans:wght@500&display=swap');
 
-body {
-  font-family: 'Work Sans';
-  font-style: normal;
-  font-weight: 600;
-  font-size: 30px;
-  line-height: 35px;
-  letter-spacing: 0.1em;
-
-  color: #000000;
+* {
+  margin: 0;
```

Comment:
if margin and padding are 0, do we even require this? What is the benefit we are getting out of this?